search.go

```diff
@@ -73,7 +73,10 @@ func searchHandlerWithProxyMode(logger *zap.Logger, indexer Indexer, proxyMode *
 	}
 }
 
-func newSearchFilterFromQuery(query url.Values) (*packages.Filter, error) {
+func newSearchFilterFromQuery(ctx context.Context, query url.Values) (*packages.Filter, error) {
+	span, _ := apm.StartSpan(ctx, "NewSearchFilterFromQuery", "app")
+	defer span.End()
```

Comment:
Nit. This span duration should be negligible, but ok if you find it useful.

Reply:
Looking again at the code of this function, I think you're right. This will not give us too much information. I'll remove it.